Packs/EDL/Integrations/EDL/README.md

```diff
@@ -1,13 +1,22 @@
 This integration provides External Dynamic List (EDL) as a service for the system indicators (Outbound feed). The feed content can then be sent to 3rd parties for improving their security alignment.
 
+## PAN-OS EDL Management to PAN-OS EDL Service migration steps
+This integration hosts the EDL on the Cortex XSOAR server. Please follow these steps to migrate your EDLs.
+1. Create IOCs in Cortex XSOAR from the EDLs data, which was previously hosted on an external web server. To do so automatically, extract your EDL as a text file from the web server, upload it as a file to the war-room and use the `ExtractIndicatorsFromTextFile` automation. e.g, `!ExtractIndicatorsFromTextFile entryID=<entry_id>` 
+2. Go to the `Indicators` and [filter](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/5-5/cortex-xsoar-admin/manage-indicators/understand-indicators/indicators-page.html#idf15421df-a0e0-41fb-b8d4-07d0f610bbec) to find all of the Indicators you extracted from the text file.
+3. If needed, batch select the indicators and add a tag to indicators you which to host as a specific EDL. Use this tag in the `Indicator Query` integration parameter when configuring the integration. 
+4. Maintain your indicators by understanding the (expiration)[https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/manage-indicators/understand-indicators/indicator-expiration.html] policy and the [exclusion](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/manage-indicators/understand-indicators/exclusion-list.html) list.
+5. You can always edit a single or bulk of indicators from the Indicators screen.
+6. Follow the rest of this guide to make sure that the PAN-OS device has connection to the EDL service.
+7. Upon a configuration of 100s of FWs and a Panorama device, we recommend creating an EDL object in the Panorama device that will be populated from the `PAN-OS EDL Service`. the EDL object should be pushed to its respective FWs. 
```

Comment:
capital T in the second sentence.
+ small rephrasing to the first one:
`Upon a configuration of 100 FWs or more, we recommend using your Panorama device and creating an EDL object there that will be populated from the...`